openapi/backoffice/openapi.yaml

```diff
@@ -685,34 +685,14 @@ definitions:
             type: array
             minItems: 1
             items:
-              $ref: '#/definitions/Address'
+              type: string
```

Comment:
I'd rather add a `minLength` to be sure that this string is a non empty string

Reply:
Added minLength to 10